lib/fluent/event.rb

```diff
@@ -308,11 +308,13 @@ def ensure_decompressed!
   end
 
   module ChunkMessagePackEventStreamer
-    # chunk.extend(ChunkEventStreamer)
+    # chunk.extend(ChunkMessagePackEventStreamer)
     #  => chunk.each{|time, record| ... }
     def each(unpacker: nil, &block)
       open do |io|
-        (unpacker || Fluent::MessagePackFactory.msgpack_unpacker(io)).each(&block)
+        # Note: If need to use `unpacker`, then implement it,
```

Comment:
It seems that we should raise `NotImplementedError` for this case to clarify it.

Reply:
Thanks for your advice!
I see! I will fix it!

Reply:
Done!